packages/server/src/controllers/UsersController.js

```diff
@@ -4,7 +4,135 @@ import UsersValidator from '../validators/UsersValidator'
 
 const { User, UserAddress } = Models
 
+// temp dummy models
+let Users = { }
+let UserEmails = { }
+let UserSessions = { }
+
+
 export default {
+  async details(req, res) {
+    let body = {
+      "emailExists": false,
+      "emailVerified": false,
+      "addressVerified": false,
+      "notificationsEnabled": false
+    }
+    Object.assign(body, Users[req.params.address])
+    res.send(body)
+  },
+
+
+  sessions: {
+    async create(req, res) {
+      if (!(req.params.address in Users)) {
+        Users[req.params.address] = {}
+      }
+      Users[req.params.address]['addressVerified'] = true
+      if (!(req.params.address in UserSessions)) {
+        UserSessions[req.params.address] = {}
+      }
+      UserSessions[req.params.address][req.session.id] = true
```

Comment:
To avoid repeating `req.params.address` every time you can extract it at the beginning of each function as follows:
```js
const { params: { address } }  = req
```

Reply:
oh nice thanks